cypress/e2e/workspace.spec.js

```diff
@@ -62,27 +62,33 @@ describe('Workspace', function() {
 		cy.get('a[href*="/apps/files/recent"]')
 			.click()
 		cy.get('#rich-workspace .ProseMirror')
-			.should('not.exist')
+			.should('not.visible')
 	})
 
 	it('adds a Readme.md', function() {
+		const url = '**/remote.php/dav/files/**'
+		cy.intercept({ method: 'PUT', url })
+			.as('addDescription')
+
 		cy.visit(`apps/files?dir=/${encodeURIComponent(currentFolder)}`)
 		cy.get('.files-fileList').should('not.contain', 'Readme.md')
-		cy.openWorkspace()
-			.type('Hello')
-			.should('contain', 'Hello')
+
+		cy.get('.files-controls').within(() => {
+			cy.get('.button.new').click()
+			cy.get('.newFileMenu a.menuitem[data-action="rich-workspace-init"]').click()
+			cy.wait('@addDescription')
+		})
+
 		openSidebar('Readme.md')
-		cy.log('Regression test for #2215')
-		cy.get('#rich-workspace .ProseMirror')
+		cy.get('#rich-workspace .text-editor .text-editor__wrapper')
 			.should('be.visible')
-			.should('contain', 'Hello')
 	})
 
 	it('formats text', function() {
 		cy.visit(`apps/files?dir=/${encodeURIComponent(currentFolder)}`)
 		cy.openWorkspace()
-			.type('Format me')
-			.type('{selectall}')
+			.type('Format me', { force: true })
```

Comment:
Would be good to have a way without force. Maybe we need to focus the workspace and wait for the menubar to appear first.